go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/IBM-Cloud/power-go-client
 
-go 1.17
+go 1.20
```

Comment:
The terraform-provider-ibm is still at 1.18.  Will it cause any conflict if we update to 1.20 instead of 1.18?
```suggestion
go 1.18
```

Reply:
Don't think that's an issue. Could you provide your approval? I will do a integration test from terraform provider tomorrow morning (IST). If it works then will merge this.

Reply:
Verified.